Classes/ScrnMK23Pistol.uc

```diff
@@ -7,45 +7,49 @@ var transient bool  bShortReload;
 var transient bool bTweeningSlide;
 var float TweenEndTime;
 var vector PistolSlideOffset; //for tactical reload
-var vector PistolSlideLockedOffset; //for tactical reload
 
 simulated function BringUp(optional Weapon PrevWeapon)
 {
-    Super.BringUp(PrevWeapon);
-    if (MagAmmoRemaining == 0)
-        LockSlideBack();
-}
-
-simulated function ResetSlidePosition()
-{
-    SetBoneLocation( 'Slide', PistolSlideOffset, 0 ); //reset slide position
+    if (Level.NetMode != NM_DedicatedServer)
+    {
+        Super.BringUp(PrevWeapon);
```

Comment:
Shouldn't super call be moved outside the dedicated server check (i.e., called on dedicated server too)?

Reply:
oops, this was a bad copypaste